### ISSUE TITLE
VideoJs styles override custom component styles

### ISSUE THREAD

**Issue by szJazzy** [videojs/video.js#6991]
## Description
I've created custom component, and it's added to videoJs. The problem is that some styles, for example `.video-js button` affects my custom button in component (which is nested insinde `.video-js`). 
![image](https://user-images.githubusercontent.com/75796267/101787552-835e5100-3aff-11eb-988c-411b4341c8ae.png)

### Expected
no general styles should be applied in videoJs (as `button`)

**Comment by gkatsev** [comment]
Yeah, our styles are often a bit too specific. Unfortunately, because the styles have been like this for a long time, it's very hard to change them without potentially breaking people.
My recommendation, which is definitely not ideal, would be to increase your styling when you're adding buttons inside the Video.js player div.

**Comment by donremf** [comment]
I use only classes to style my component, but `videojs` adds styles to type `button`. It would be better to add a class e.g `video-js__button` and style it - so it won't affect any element created inside.
